### PR TITLE
Cloud hosted Guides blog post

### PR DESCRIPTION
**Pull request by jakub-pomykala**
Hi everyone,

If you could please review the guide here for the final time.

The draft guide can be found [here](https://blogs-draft-openlibertyio.mybluemix.net/blog/2021/07/22/cloud-hosted-guides-post.html)

Latest screenshot of the guide below incase the site is down:

image

### CODE REVIEW

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Interactive developer guides running in the cloud"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jamiecoleman92
+author_github: https://github.com/GITHUB_USERNAME
+seo-title: Interactive developer guides running in the cloud - OpenLiberty.io
+seo-description: Open Liberty now provides interactive guides (hosted in the cloud) for developers and users to learn about cloud-native Java API and technologies like MicroProfile and Jakarta EE without requiring any local prerequisites and setup.
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
seo-description: Open Liberty now provides interactive guides (tutorials hosted in the cloud) for developers and users to learn about cloud-native Java API and technologies like MicroProfile and Jakarta EE without requiring any local prerequisites and setup.
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Interactive developer guides running in the cloud
+Jamie Coleman <https://github.com/jamiecoleman92>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Our team of engineers have been hard at work creating new content for developers to enjoy. We now have 57 different guides on a range of topics covering development, build, testing and deployment of cloud-native Java applications and microservices. They are designed to help developers get started quickly with learning the technologies and Open Liberty. Excitingly, we now offer the possibility to run our guides in a hosted environment without having to worry about local setup and prerequisites. You can access these cloud-hosted guides by going to our link:{url-prefix}/guides/[guides] page, select a guide, and clicking the “Run in cloud” button in the top right as shown in the following image:
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
Our team of engineers have been hard at work creating new content for developers to enjoy. We now have 57 different guides on a range of topics covering development, build, testing and deployment of cloud-native Java applications and microservices. They are designed to help developers get started quickly with learning various Java and cloud technologies and Open Liberty. Excitingly, we now offer the possibility to run our guides in a hosted environment without having to worry about local setup and prerequisites. You can access these cloud-hosted guides by going to our link:{url-prefix}/guides/[guides] page, select a guide, and clicking the “Run in cloud” button in the top right as shown in the following image:
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+image::/img/blog/OL_Cloud-hosted-1.png[Image 1,width=70%,align="center"]
+
+== What guides are available?
+
+Now we have gone live we have 22 guides ready for you to try out:
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
We are now live with 22 guides you can try out in the cloud:
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+* link:{url-prefix}/guides/microprofile-opentracing.html[Enabling distributed tracing in microservices with Zipkin microprofile-health.]
+* link:{url-prefix}/guides/microprofile-health.html[Adding health reports to microservices]
+* link:{url-prefix}/guides/microshed-testing.html[Testing a MicroProfile or Jakarta EE application with MicroShed Testing.]
+* link:{url-prefix}/guides/reactive-service-testing.html[Testing reactive Java microservices with MicroShed Testing.]
+* link:{url-prefix}/guides/containerize.html[Containerizing microservices.]
+* link:{url-prefix}/guides/kubernetes-intro.html[Deploying microservices to Kubernetes.]
+
+== How to get started
+
+The portal is quite straight forward to navigate so once you have clicked on the “Run in cloud” button mentioned above you will be taken to a login page if you do not have an account or are not logged in. From here you can login or create a new account and we only require very limited information to get started. You can also use social logins to create an account and then use that later to login quickly.
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
It is easy to get started once you have clicked on the “Run in cloud” button mentioned above. You will be taken to a login page for the cloud environment. From here you can login or create a new account and we only require minimal information for you to get going. You can also use social logins to create an account and then use that later to login quickly. 
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+The portal is quite straight forward to navigate so once you have clicked on the “Run in cloud” button mentioned above you will be taken to a login page if you do not have an account or are not logged in. From here you can login or create a new account and we only require very limited information to get started. You can also use social logins to create an account and then use that later to login quickly. 
+
+image::/img/blog/OL_Cloud-hosted-3.png[Image 3,width=70%,align="center"]
+
+Once you are logged in you will then be directed to the landing page for the guide you wish to learn. Simply click on the “Access cloud-hosted guide” button and after a few minutes your environment should be set-up and ready to use.
+
+== What is the cloud environment and why are we using it with our guides?
+
+The cloud environment is a learning platform that runs through a user’s browser. It runs in containers that are deployed to OpenShift in the IBM Cloud. This allows us not only to teach users how to build great cloud-native applications but also demonstrates that in practice with how the environment is deployed.
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
The cloud environment we are using for these interactive guides is a learning platform that runs through a user’s browser. It runs in containers that are deployed to OpenShift in the IBM Cloud. This allows us not only to teach users how to build great cloud-native applications but also demonstrates that in practice how one interacts with the deployment environment. 
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+== What is the cloud environment and why are we using it with our guides?
+
+The cloud environment is a learning platform that runs through a user’s browser. It runs in containers that are deployed to OpenShift in the IBM Cloud. This allows us not only to teach users how to build great cloud-native applications but also demonstrates that in practice with how the environment is deployed. 
+
+The hosted environment is provided by IBM Skills Network. These interactive guides won't be possible without the support and help of the IBM SKills Network team. A shoutout and thank you to them!
+
+image::/img/blog/OL_Cloud-hosted-2.png[Image 2,width=70%,align="center"]
+
+This environment gives users access to all the tools required to build cloud-native applications such as Docker, Kubernetes, OpenShift, Maven, Gradle, OpenJ9 JVM and the Open Source Theia IDE.
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
This environment gives users access to all the tools required to build cloud-native applications such as Docker, Kubernetes, OpenShift, Maven, Gradle, OpenJDK with Eclipse OpenJ9 VM, the open-source Eclipse Theia IDE, and of course, Open Liberty.
```

File: posts/2021-07-22-cloud-hosted-guides-post.adoc [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+== Our cloud environment - Skills Network
```
Comment (yeekangc) [inline posts/2021-07-22-cloud-hosted-guides-post.adoc]:
```suggestion
== Using Open Liberty guides in the cloud
```

**Review by yeekangc — CHANGES_REQUESTED** [review]

**Review by jamiecoleman92 — DISMISSED** [review]
Looks all good to me and thank you for the review and changes!

**Review by dmuelle — DISMISSED** [review]

**Comment by yeekangc** [comment]
Thanks all!  LGTM and approving.

I feel we can add a sentence and ask users to leave us feedback re their experience using the guides too, @jakub-pomykala @jamiecoleman92.

**Review by yeekangc — DISMISSED** [review]

**Comment by dmuelle** [comment]
@yeekangc - w/r/t feedback- should we point them to our mailing list (as in regular blog posts) or is there something intrinsic to the cloud hosted guides where they can respond?

**Review by yeekangc — APPROVED** [review]
Thanks, @dmuelle.

**Comment by yeekangc** [comment]
We ran into issues with the SNL URLs.  Please hold off publishing until that is fixed and tested.  @gkwan-ibm can provide the green light.  We can target this post for Monday.  Thank you all.

**Pull request merged**